### PR TITLE
doc: Create ADOPTERS.md

### PR DESCRIPTION
**Pull request by sougou**
Addresses #2789 

@adkhare @harshit-gangal @bbeaudreault @derekperkins @Rastusik @demmer @tirsen @rnavarro 
Let me know if you want anything changed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,11 +17,13 @@ since 2011, and has grown to encompass tens of thousands of MySQL nodes.
 
 For more about Vitess, please visit [vitess.io](http://vitess.io).
 
+Vitess has a growing community. You can view the list of adopters [here](ADOPTERS.md).
```
Comment (michael-berlin) [inline README.md]: A technical comment here: This should be an absolute link.

Our "base" image copies the content of this file and shows it on Docker Hub: https://hub.docker.com/r/vitess/base/
The relative link there won't work.

Comment (sougou) [inline README.md]: Fixed. We should work on getting this exposed on vitess. io. But I was thinking the work there may have to be more fancy, with logos, etc.

File: ADOPTERS.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+This is an alphabetical list of known adopters of Vitess. Some have already gone into production, and others are at various stages of testing.
+
+* [BetterCloud](http://bettercloud.com)
+* [FlipKart](http://flipkart.com)
+* [HubSpot](http://hubspot.com)
```
Comment (bbeaudreault) [inline ADOPTERS.md]: Can you link to http://product.hubspot.com/ instead? That's the landing page for our dev team blog and such.

**Comment by bbeaudreault** [comment]
LGTM

**Comment by rnavarro** [comment]
LGTM

**Comment by demmer** [comment]
LGTM

**Comment by alainjobart** [comment]
LGTM

**Pull request merged**